src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const states = [];
+  const newState = { ...state };
+
+  actions.forEach((action) => {
+    if (action.type === 'addProperties') {
```

Comment:
Use destruction for `action`
```suggestion
    const { type, ....... } = action;

    if (type === 'addProperties') {
```